bigbluebutton-html5/imports/api/deskshare/server/handlers/incomingDeskshareEvent.js

```diff
@@ -1,14 +1,17 @@
+import Deskshare from '/imports/api/deskshare';
 import Meetings from '/imports/api/meetings';
+import modifyDeskshareStatus from '../modifiers/modifyDeskshareStatus';
 
-export function handleIncomingDeskshareMessage(arg) {
+export default function incomingDeskshareEvent(arg) {
```

Comment:
Forgot to `check` the parameters of the payload and you can extract the payload right on the function parameters `({ payload })` take a look on the other api handlers.